types/react-table/index.d.ts

```diff
@@ -176,6 +176,7 @@ export type UseTableOptions<D extends object> = {
     getSubRows: (originalRow: D, relativeIndex: number) => D[];
     getRowId: (originalRow: D, relativeIndex: number, parent?: Row<D>) => string;
     autoResetHiddenColumns: boolean;
+    autoResetResize: boolean;
```

Comment:
I think that this should have been added to `UseResizeColumnsOptions`, since it should really only be used if that hook in in use.

Reply:
thanks for the comment, fixed it